### DISCUSSION TITLE
nAN Values

### DISCUSSION THREAD

**Discussion by HyakuzaO** [georgedouzas/sports-betting#62] [Q&A]
![imagem](https://github.com/georgedouzas/sports-betting/assets/160853868/c0927327-b013-44aa-8a61-8079f08f4f91)
Im using the Cli command, everything works fine but when it save the value bets it returns like the image. anyone can help me ? 

Best regards to all

**Comment by georgedouzas** [comment]
Hi @HyakuzaO,

Please be more specific about the commands you use.

**Reply by HyakuzaO** [reply]
![image](https://github.com/georgedouzas/sports-betting/assets/160853868/6c1fe64f-13df-4458-a2d7-5894bc5cecf4)

Im using the comands that you provide, it create all csv files, x train etc , but when i run and save the value bets the csv file only shows that tables, dont show what game got value bet etc

**Reply by HyakuzaO** [reply]
@georgedouzas What am i doing wrong?🙏
